design/iterations.go

```diff
@@ -85,6 +88,24 @@ var _ = a.Resource("iteration", func() {
 		a.Response(d.InternalServerError, JSONAPIErrors)
 		a.Response(d.Unauthorized, JSONAPIErrors)
 	})
+	a.Action("update", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.PATCH("/:id"),
+		)
+		a.Description("update the iteration with the given id.")
```

Comment:
fixed in https://github.com/almighty/almighty-core/pull/652/commits/6a004852b083521ec2d390e83dccfadc911969f0